ui/package.json

```diff
@@ -8,15 +8,15 @@
         "node": ">=18.0.0"
```

Comment:
Something for later: this file declares `engines.node` but `apps/platform/package.json` does not.

From a description of it https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines and [engine-strict](https://docs.npmjs.com/cli/v9/using-npm/config#engine-strict) it might be useful to catch dependencies with different node version requirements that may potentially cause problems.